bin/htmlhint

```diff
@@ -9,7 +9,7 @@ var glob = require("glob");
 var parseGlob = require('parse-glob');
 var request = require('request');
 
-var HTMLHint  = require("../dist/htmlhint.js").default;
+var HTMLHint  = require("../dist/htmlhint.js").hint;
```

Comment:
```suggestion
var HTMLHint  = require("../dist/htmlhint.js").HTMLHint;
```
I think you miss this.

Reply:
Yes, I was just trying to find where the issue was. Thanks Nicolas!

Reply:
Just did the fix @nicolashenry